app.js

```diff
@@ -8,6 +8,21 @@ const app = express();
 // app.use(bodyParser.urlencoded()); //  x-www-form-urlencoded Use with form <form>
 app.use(bodyParser.json()); //  application/json
 
+// CORS
+app.use((req, res, next) => {
+  //  Allow origin and can access all clients using wildcard
+  res.setHeader('Access-Control-Allow-Origin', '*');
```

Comment:
We're using a wildcard to allow all domains that's reached our server but this `'*'` can be a specific domain